### DISCUSSION TITLE
Black Icon on ExtraActionButton When Masque is Enabled

### DISCUSSION THREAD

**Discussion by Tonyleila** [SFX-WoW/Masque#180] [Issues] [answered]
https://i.imgur.com/lYz3f5m.jpg

Using dominos. The screenshot show a framestack maybe it helps :)

**Comment by StormFX** [comment]
Looks to me like the Icon is being placed below the Backdrop. Masque puts the Icon above it, so something else must be affecting it. I'll look into it.

**Comment by Neithis** [comment]
I have also noticed this issue, was driving me mad to figure out what it was!

You can start to see the icon if you reduce the opacity on of the extra button on Dominos

Edit - Add Images
[Full Opacity](https://prnt.sc/kn43y8)
[Reduced Opacity on Dominos](https://prnt.sc/kn4688)

**Comment by StormFX** [comment, marked as answer]
I've tried to reproduce this and can't. The Extra Action Bar/Button works fine on my side, with no black overlay at all. Given that these are the only two reports I've had of it, I'm guessing you've another add-on installed that's causing it or something else is going on.

You'll need to back up your saved variables files for Domino and Masque and then remove them. Then start the game with ONLY Dominos and Masque enabled. To be sure it's not skin-related, don't use any extra skins.

**Comment by StormFX** [comment]
Closing since I'm unable to reproduce this and it's been awhile.
